python/ray/tests/test_failure_3.py

```diff
@@ -155,7 +154,7 @@ def pid(self):
     rs = []
 
     def submit():
-        for i in range(100000):
+        for i in range(10000):
```

Comment:
should we keep this?

Reply:
I tested it and I notice it's going to run for such a long time before it terminated. So I reduce the size.